src/models/constants/JobTypes.js

```diff
@@ -3,6 +3,7 @@ const JobTypes = Object.freeze([
     "PART-TIME",
     "SUMMER INTERNSHIP",
     "CURRICULAR INTERNSHIP",
+    "GRANT",
```

Comment:
Maube this should be Research grant. Are there any other ones that im missing ?

Reply:
I don't think so